packages/react-bindings/src/styles/getStyles.ts

```diff
@@ -42,81 +43,90 @@ export type GetStylesResult = {
   theme: StylesContextValue['theme']
 }
 
+const variablesCache = new WeakMap<ThemePrepared, Record<string, any>>()
+
+const resolveVariables = (
+  displayName: string,
+  theme: ThemePrepared,
+  variables: any | undefined,
+): any => {
```

Comment:
Oops, let's fix any there:

```suggestion
): ComponentVariablesObject => {
```